sdk/servicebus/service-bus/test/perf/track-2/sbBase.spec.ts

```diff
@@ -9,24 +9,22 @@ import * as dotenv from "dotenv";
 dotenv.config();
 
 const connectionString = getEnvVar("SERVICEBUS_CONNECTION_STRING");
-const sbClient = new ServiceBusClient(connectionString);
 
 export abstract class ServiceBusTest<TOptions> extends PerfStressTest<TOptions> {
-  sbClient: ServiceBusClient;
+  static sbClient: ServiceBusClient = new ServiceBusClient(connectionString);
```

Comment:
Any reason not to do this in globalSetup()?

Reply:
Doesn't matter. It would work the same way.

In my mind.. I've kept globalSteup for creating the resources and the initialization of clients at the static variable declaration itself.